web-ui/src/main/resources/catalog/templates/editor/import.html

```diff
@@ -6,6 +6,10 @@ <h1 data-translate="">ImportRecord</h1>
     </div>
   </div>
 
+  <div class="alert alert-warning" data-ng-show="groups.length === 0"
```

Comment:
When can this case occur?

Reply:
When a userAdmin is not editor or reviewer in any group. UserAdmin can access the page, try to load a record and will get an error.

Reply:
Ok, clear. This seem different as in 3.12.x, where UserAdmin can import the record, what is bizarre anyway.